backend/log/log.go

```diff
@@ -12,10 +12,11 @@ type Logger interface {
 	Error(msg string, args ...interface{})
 }
 
-// New creates a new logger.
-func New() Logger {
+// NewLoggerWithName creates a named logger
+func NewLoggerWithName(name string) Logger {
 	return &hclogWrapper{
 		logger: hclog.New(&hclog.LoggerOptions{
+			Name: name,
```

Comment:
I don't know if this is necessary or not -- is it used?

All the existing (v0) plugins set a named logger and wanted to set the level from environment variables...

Reply:
No, not needed. The plugin id is already set in logger in Grafana.

Reply:
```suggestion
```